scripts/run-in-docker.sh

```diff
@@ -22,22 +22,22 @@ export AWS_SECRET_ACCESS_KEY=$(aws configure get aws_secret_access_key --profile
 # We are piping to stdin of read command, which lets us set a variable
 # look up read docs - read for linux, cat for mac
 #read -r -d '' SCRIPT <<'EOF'
-# cat <<'EOF'
-# const testPostBody = {
-#     id: "Sally"
-# }
-
-# // This is ignored if we use a GET request
-# const testEvent = { 
-#     httpMethod: POST,
-#     body: JSON.stringify(testPostBody)
-# }
-# console.log(JSON.stringify(testEvent))
-# EOF
-
+SCRIPT=$(cat <<'EOF'
+const testPostBody = {
+    id: "Sally"
+}
+
+// This is ignored if we use a GET request
+const testEvent = { 
+    httpMethod: "GET",
```

Comment:
Then you can do as you expect:
```suggestion
    httpMethod: "${METHOD}",
```